src/components/DatePicker/styles.css

```diff
@@ -1,4 +1,4 @@
-#datePickerMonth {
+#DatePicker select {
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
```

Comment:
In the sense of multiple instances on a single page causing WCAG issues, so would having multiple `DatePicker` instances, right? Could this be changed to a class like `.datePicker` instead?

Reply:
sure, makes sense. Will do.